src/styles/starts.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+  &--1 :nth-child(-n + 1),
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
```

Comment:
The selectors for the modifier classes (--1, --2, etc.) should be formatted with a space before the colon to follow the code style guidelines. For example, it should be '&--1 :nth-child' instead of '&--1:nth-child'.